File: src/me/krickl/memebotj/InternalCommands/QuitCommand.java
package me.krickl.memebotj.InternalCommands;

import me.krickl.memebotj.ChannelHandler;
import me.krickl.memebotj.CommandHandler;
import me.krickl.memebotj.UserHandler;

public class QuitCommand extends CommandHandler {

	public QuitCommand(String channel, String command, String dbprefix) {
		super(channel, command, dbprefix);
		this.setAccess("botadmin");
		this.setNeededCommandPower(75);
		this.setUnformattedOutput("You have been terminated MrDestructoid");
		this.setHelptext("Quits the bot");
	}

	@Override
	public void commandScript(UserHandler sender, ChannelHandler channelHandler, String[] data) {
		System.exit(0);
	}

}

File: src/me/krickl/memebotj/InternalCommands/QuitCommand.scala
package me.krickl.memebotj.InternalCommands

import me.krickl.memebotj.ChannelHandler
import me.krickl.memebotj.CommandHandler
import me.krickl.memebotj.UserHandler

class QuitCommand(channel: String, command: String, dbprefix: String) extends CommandHandler(channel,
	command, dbprefix) {

	this.setAccess("botadmin")

	this.setNeededCommandPower(75)

	this.setUnformattedOutput("You have been terminated MrDestructoid")

	this.setHelptext("Quits the bot")

	override def commandScript(sender: UserHandler, channelHandler: ChannelHandler, data: Array[String]) {
		System.exit(0)
	}
}
